### PR TITLE
Update schema

### PR DESCRIPTION
**Pull request by ilfa**
- update documentation
- make `IpLocation` field and all its properties optional

### CODE REVIEW

File: res/fingerprint-server-api.yaml [hunk 1]
```diff
@@ -24,22 +30,41 @@ servers:
...
   - ApiKeyQuery: []
+x-readme:
+  samples-languages:
+    - curl
+    - node
+    - php
+    - python
+    - csharp
+    - java
+    - go
```
Comment (TheUnderScorer) [inline res/fingerprint-server-api.yaml:41]: What does this do?

File: template/api.mustache [hunk 2]
```diff
@@ -25,21 +25,21 @@ namespace {{packageName}}.{{apiPackage}}
                 /// {{summary}}
                 /// </summary>
             /// <remarks>
-                /// {{notes}}
+                /// {{{notes}}}
```
Comment (TheUnderScorer) [inline template/api.mustache:28]: Why do we need three "{"? 😁

Comment (ilfa) [reply template/api.mustache:28]: To correctly render backtick symbol (&#x60;) without replacing it to `&#x60;`

**Review by TheUnderScorer — APPROVED** [review]

**Pull request merged**